tests/test_multi_maintenance.py

```diff
@@ -203,6 +203,22 @@ def test_008b_node2_to_maintenance():
     node3.check_synchronous_standby_names(ssn)
 
 
+def test_008a_stop_primary():
+    # node3 is the current primary
+    assert node3.get_state().assigned == "primary"
+    node3.fail()
+
+    # check that even after 30s node3 is still not set to draining
+    node3.sleep(30)
+    assert not node3.get_state().assigned == "draining"
+    assert node3.get_state().assigned == "primary"
+
+
+def test_008a_start_primary():
+    node3.run()
+    assert node3.wait_until_state(target_state="primary")
+
+
```

Comment:
Can we add a test that `enable maintenance --allow-failover` fails here?

Reply:
Added, thanks for catching it!